nixos/modules/services/web-servers/nginx/default.nix

```
@@ -472,7 +472,7 @@ let
 
   mkCertOwnershipAssertion = import ../../../security/acme/mk-cert-ownership-assertion.nix;
 
-  oldHTTP2 = versionOlder cfg.package.version "1.25.1";
+  oldHTTP2 = (versionOlder cfg.package.version "1.25.1" && !(cfg.package.pname == "angie" || cfg.package.pname == "angieQuic"));
```

I'm not sure how openresty or tengine handles this, but neither seems to share the version with nginx. Would this make sense? CC @RaitoBezarius

```suggestion
  oldHTTP2 = (versionOlder cfg.package.version "1.25.1" && (cfg.package.pname == "nginx" || cfg.package.pname == "nginxQuic"));
```

---

The suggested variant doesn't work.
I tested my variant with other packages - `openresty` and `tengine`.